src/cordova/platform.js

```diff
@@ -1,769 +0,0 @@
-/**
```

Comment:
to confirm, you ripped out this file into multpile smaller files. I like it. It has been replaced by addHelper.js, check.js, getPlatformDetailsFromDir.js, index.js, list.js, remove.js and save.js

I see tests for addHelper. Do the other modules need new unit tests?